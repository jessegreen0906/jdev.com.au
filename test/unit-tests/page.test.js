import Page from '../../src/components/page';
import React from 'react';
import renderer from 'react-test-renderer';

test('should render', function () {
	const component = renderer.create(
		<Page />,
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});

// test('Expect the obvious', function() {
// 	expect(1).toEqual(1);
// });