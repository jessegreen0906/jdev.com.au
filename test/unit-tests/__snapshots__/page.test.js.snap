// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<h1>
  Heading
</h1>
`;
